classes/search/ArticleSearchDAO.inc.php

```diff
@@ -80,8 +80,12 @@ public function getPhraseResults($journal, $phrase, $publishedFrom = null, $publ
 				JOIN publication_settings ps ON (ps.publication_id = p.publication_id AND ps.setting_name=\'issueId\' AND ps.locale=\'\')
 				JOIN issues i ON (CAST(i.issue_id AS CHAR(20)) = ps.setting_value AND i.journal_id = s.context_id)
 				JOIN submission_search_objects o ON (s.submission_id = o.submission_id)
+				JOIN journals con ON con.journal_id = s.context_id
+				LEFT OUTER JOIN journal_settings con_s_publishingMode on con.journal_id = con_s_publishingMode.journal_id AND con_s_publishingMode.setting_name = \'publishingMode\'
```

Comment:
This is only a very minor comment, but:
I find easier readable short table aliases -- I also think we use them generally where possible -- thus I would prefer to have `journals j` and `journal_settings js`. For OPS main branch: for servers maybe `servers j` or `servers c`
Also, we mostly use only LEFT JOIN (without OUTER) and write `ON` (instead of `on`).

Reply:
1. Changed the aliases to `j` and `js` for all changes.
2. Removed `OUTER` from `LEFT OUTER JOIN`
3. Changed `on` to `ON`